src/calls-cost.js

```diff
@@ -20,6 +20,32 @@
  * @property {ProcessedCall[]} callsDetails - Processed information about calls
  */
 
+function calculate(duration, additionalCost, firstminutes = 0, cost = 0) {
+  let amount = Math.min(duration, firstminutes) * cost;
+  if (duration > firstminutes) {
+    amount += (duration - firstminutes) * additionalCost;
+  }
+  return amount;
+}
+function ProcessedCall(call) {
+  let amount = 0;
+  const { duration } = call;
+  switch (call.type) {
+    case 'International':
+      amount = calculate(duration, 3.03, 3, 7.56);
+      break;
+    case 'National':
+      amount = calculate(duration, 0.48, 3, 1.2);
+      break;
+    case 'Local':
+      amount = calculate(duration, 0.2);
+      break;
+    default:
+      return null;
+  }
+  return amount;
```

Comment:
Here, you could've saved some lines.

You can return directly from switch cases:

```ts
case 'International':
      return calculate(duration, 3.03, 3, 7.56);

...other cases

default:
     retturn null; 
```